nxrm-ha-helm/README.md

```diff
@@ -18,6 +18,119 @@
 This Helm chart configures the Kubernetes resources that are needed for a high-availability (HA) Nexus Repository deployment on AWS/Azure/Onprem.
 
 ---
+
+# Pre-requisites
+
+### Storage
+The default configuration uses an emptyDir volume for storing Nexus Repository logs. However, this is only for demonstration purposes. For production, we strongly recommend that
+you configure dynamic provisioning of persistent storage or attach dedicated local disks based on your deployment environment as explained below.
+
+#### Cloud deployments (AWS/Azure)
+* Ensure the appropriate Container Storage Interface (CSI) driver(s) are installed on the Kubernetes cluster on your chosen cloud deployment. Please refer to AWS EKS/Azure AKS documentation for details on configuring CSI drivers.
+
+#### On-premises deployments
+1. Attach separate disks (i.e., separate from the root disk) to your worker nodes.
+2. Install the Local Persistence Volume Static Provisioner. Please refer to [Local Persistence Volume Static Provisioner](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner) documentation.
+3. Use the Local Persistence Volume Static Provisioner to automatically create persistent volumes for your chosen storage class name as documented [here](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner)
+
+#### Configuring for dynamic persistent volume provisioning
+* Set the `storageClass.name` parameter to a storage class name. This could be one of the default storage classes automatically created in your managed Kubernetes cluster on your chosen cloud (e.g., if you're using AWS EKS) or one that you would like to create.
```

Comment:
```suggestion
* Set the `storageClass.name` parameter to a storage class name. This could be one of the default storage classes automatically created in your managed Kubernetes cluster on your chosen cloud (e.g., if you're using AWS EKS) or one that you would like to create:
```